### PR TITLE
Docs: add disable_brute_force_login_protection to configuration document

### LINKED ISSUE #20287: Docs: Option disable_brute_force_login_protection missing in config docs
In grafana v6.4.x the property GF_SECURITY_DISABLE_BRUTE_FORCE_LOGIN_PROTECTION is not used, is there a substitution for this property or it is definitely obsolete?

### PR DESCRIPTION
**Pull request by yzdann**
Fixes: #20287
I was wondering if there is a way to prevent a brute force attack so I first looked at the documentation and found nothing. After searching more I found this feature in one PR. For more clarification, I add this to documentation.

### CODE REVIEW

**Comment by marefr** [comment]
Thanks. Please sign the CLA

**Comment by yzdann** [comment]
I signed the CLA but it seems I forgot to add `.gitconfig` and I used my global git config email address for this commit. Should i close this PR and open new PR with my GitHub valid email!? or there is a simpler way?

**Comment by yzdann** [comment]
I amend and push again.

File: docs/sources/installation/configuration.md [hunk 1]
```diff
@@ -355,6 +355,11 @@ Define a whitelist of allowed IP addresses or domains, with ports, to be used in
 
 Set to `true` if you host Grafana behind HTTPS. Default is `false`.
 
+
```
Comment (aknuds1) [inline docs/sources/installation/configuration.md:358]: Do you mind just removing this extra blank line, for consistency? It reads better without.
```suggestion
```

**Review by aknuds1 — APPROVED** [review]

File: docs/sources/installation/configuration.md (hunk 1 above)
Comment (yzdann) [reply docs/sources/installation/configuration.md]: Thank you and sorry for that.

**Comment by aknuds1** [comment]
Thank you for helping to make Grafana better!

**Pull request merged**